announcements/1.14.0.md

```diff
@@ -0,0 +1,87 @@
+| cs:~anbox-charmers/ams-load-balancer | [ams-load-balancer](https://charmhub.io/ams-load-balancer) |
+| cs:~anbox-charmers/aar | [aar](https://charmhub.io/aar) |
+
+Please have a look at the [upgrade instructions](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) for further information on how migrate to Charmhub.
+
+### Anbox images based on Ubuntu 22.04
+
+Next to the existing Anbox images based on Ubuntu 18.04 Anbox Cloud now also provides Ubuntu 22.04 based ones. From an Anbox functionality perspective there is no difference, the image is just sitting on top of a newer Ubuntu version providing access to newer software from the Ubuntu archive. The Ubuntu 18.04 images are still going to be maintained going forward but will be deprecated at some point in the future (before Ubuntu 18.04 EOL in April 2023).
+
+### Hook support for AMS applications
+
+[Applications](https://discourse.ubuntu.com/t/manage-applications/24333) in AMS have now support for hooks. Hooks were until now only present as part of [addons](https://discourse.ubuntu.com/t/managing-addons/17759) and are now fully supported with the same feature set by applications.
+
+Adding hooks as part of [applications](https://discourse.ubuntu.com/t/manage-applications/24333) allows in some cases for better structure when functionality does not need to be shared with other applications.
+
+You can find more information [here](tbd)
+
+### Vulkan support for AMD and Intel GPUs
+
+For supported AMD and Intel GPUs Vulkan support is now available and allows regular Android applications making use of the Vulkan graphics API to be used.
+
+### Multi node support for the Anbox Cloud Appliance (experimental)
+
+The Anbox Cloud Appliance has been meant to be used on a single machine so far but with the 1.14 release experimental support to join the initial machine with additional ones for increased capacity is added. Multi node support does not provide high availability functionality at this time and is purely meant to extend capacity of the underlying LXD cluster.
+
+You can find more information [here](tbd)
+
+### Shiftfs disabled by default
+
+Until 1.14 Anbox Cloud enabled [shiftfs](https://discuss.linuxcontainers.org/t/trying-out-shiftfs/5155) support in LXD by default. Shiftfs allows for instant creation of containers as no manual shifting of file/directory ownership for extended use of user namespaces is necessary. However, due to bugs which affect the represented file path in `/proc/*/maps` files which some Android applications (and Android itself) run into bugs. The application use the information `/proc/*/maps` to find a path to certain libraries or memory segments they need to access. As shiftfs is being deprecated and the replacement ([idmapped mounts](https://lore.kernel.org/lkml/20210213130042.828076-1-christian.brauner@ubuntu.com/T/#u)) is not ready for the ZFS filesystem Anbox Cloud will turn off shiftfs by default starting with 1.14.0.
```

Comment:
```suggestion
Until 1.14, Anbox Cloud enabled [shiftfs](https://discuss.linuxcontainers.org/t/trying-out-shiftfs/5155) support in LXD by default. Shiftfs allows for instant creation of containers, because no manual shifting of file/directory ownership for extended use of user namespaces is necessary. However, there are bugs that affect the represented file path in `/proc/*/maps` files, which some Android applications (and Android itself) use. The applications use the information in `/proc/*/maps` to find a path to certain libraries or memory segments they need to access. 

As shiftfs is being deprecated and the replacement ([idmapped mounts](https://lore.kernel.org/lkml/20210213130042.828076-1-christian.brauner@ubuntu.com/T/#u)) is not ready for the ZFS file system, Anbox Cloud turns off shiftfs by default starting with 1.14.0.
```

Reply:
Can you check the text again? I didn't understand the third sentence (I think it's missing some parts).